### ISSUE TITLE
 include the toolchain for git books (https://www.gitbook.com/) ?

### ISSUE THREAD

**Issue by jcolomb** [OpenScienceMOOC/Main#4]
probaly good to have each module developed separately as chapter of a book, 
is it what you propose rutger?

**Comment by rvosa** [comment]
Yup, exactly. For reference, here's how we did the directory and file layout: https://github.com/pfern/OSODOS/

**Comment by wolass** [comment]
I'd suggest using the bookdown package to get a printable and online versions out of rMarkdown files. 
check it out here : https://bookdown.org/yihui/bookdown/

**Comment by Protohedgehog** [comment]
Fantastic. Does someone want to start a new section/file outline ideas for post-production and communication..?

**Comment by Protohedgehog** [comment]
Can we transfer all of these ideas here? https://github.com/OpenScienceMOOC/Main/blob/master/communication_strategy.md
